LifeCycleAssessment_Engine/LifeCycleAssessment_Engine.csproj

```diff
@@ -82,6 +82,9 @@
     <Folder Include="Modify\" />
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <PropertyGroup>
+    <PostBuildEvent>C:\Windows\System32\xcopy "$(SolutionDir)DataSets\*.*" "$(Appdata)\BHoM\DataSets" /Y /I /E</PostBuildEvent>
+  </PropertyGroup>
```

Comment:
This should not be necessary, `BHoM_UI` handles moving of datasets within toolkits when building as part of the rebuild process. This could cause later conflicts with CI installer builds so can we revert this change please.